sidecar/sidecar.go

```diff
@@ -68,15 +66,9 @@ func Main() {
 }
 
 func sidecarCommand(context *cli.Context) error {
-	if os.Getenv("A8_CHILD") == "" {
-		fmt.Println(os.Args[1:])
-		cmd := exec.Command("a8sidecar", os.Args[1:]...)
-		cmd.Stdout = os.Stdout
-		cmd.Stderr = os.Stderr
-		cmd.Env = append([]string{"A8_CHILD=yes"}, os.Environ()...)
-		// listen for SIGCHLDs from any of the supervised processes and any orphans spun off from those
-		go supervisor.ReapZombies()
-		return cmd.Run()
+	// when PID=1, launch new sidecar process and assume init responsibilities
+	if os.Getpid() == 1 {
```

Comment:
Should this be an if-else condition, so that we have a distinct branches for being PID 1 or not?